### ISSUE TITLE
In creative mode, cannot destroy travelnet box.

### ISSUE THREAD

**Issue by spyjoshx** [Sokomine/travelnet#33]
I'm in creative mode in minetest 0.5.0-dev. I placed a travelnet box and configured it, but decided to move it. But now I can't dig it. Is this a bug, or intended behavior? Or am I just doing something wrong?

**Comment by Sokomine** [comment]
It turned out to be more difficult to distinguish between punches for updates and punches for digging. In the meantime, a diamond pick was necessarry for digging travelnet stations.

Current solution is: There's no way to dig them - but a button where you can click and get the travelnet station back into your inventory.

**Comment by HybridDog** [comment]
Can you update the travelnet automatically when rightclicking it?

**Comment by Sokomine** [comment]
No. The travelnets store their formspec in metadata. They can be accessed very quickly by the client from locally stored data without having to query the server first. The locked chests in MTG that became private, hid their content from other players and thus had to send the formspec on rightclick where very slow on servers and did cost me considerable nerves. I don't want something that slow and annoying for the travelnets.
